src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.csproj

```diff
@@ -27,6 +32,18 @@
   <ItemGroup>
     <Compile Include="System\Diagnostics\StackFrameExtensions.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">  
+    <Compile Include="System\Diagnostics\StackTraceSymbols.CoreCLR.cs" />  
```

Comment:
@weshaggard This project compiles for three target groups.  I think we only need this for .NET Core (since CoreCLR will use it) but can you confirm this is the right thing to do?